examples/server.js

```diff
@@ -89,25 +89,21 @@ var sttAuthService = new AuthorizationV1(
     {
       username: process.env.SPEECH_TO_TEXT_USERNAME, // or hard-code credentials here
       password: process.env.SPEECH_TO_TEXT_PASSWORD,
-      iam_apikey: process.env.SPEECH_TO_TEXT_IAM_APIKEY // if using an RC service
+      iam_apikey: process.env.SPEECH_TO_TEXT_IAM_APIKEY, // if using an RC service
+      url: process.env.SPEECH_TO_TEXT_URL ? process.env.SPEECH_TO_TEXT_URL : SpeechToTextV1.URL
     },
     vcapServices.getCredentials('speech_to_text') // pulls credentials from environment in bluemix, otherwise returns {}
   )
 );
 app.use('/api/speech-to-text/token', function(req, res) {
-  sttAuthService.getToken(
-    {
-      url: SpeechToTextV1.URL
```

Comment:
This is the parameter that you want to modify using `process.env.SPEECH_TO_TEXT_URL ? process.env.SPEECH_TO_TEXT_URL : SpeechToTextV1.URL`

Reply:
By the way I have similar fix for the python server I made but funny enough the Watson Developer Cloud Python SDK does not support passing-in IAM keys for Authorization. I've opened an issue for this [here](https://github.com/watson-developer-cloud/python-sdk/issues/651). I will wait on them to update but in general the python server is broken anyhow (at least on MacOS) due to missing dependencies, I will include the full fix for that in my commit. If you prefer that I commit this code now (it is aligned with the expected API call I mentioned in the issue for the Python SDK) with this PR just let me know.